aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -1,6 +1,145 @@
 spec aptos_framework::account {
+    
+    spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+    }
+
     spec create_signer {
-        // TODO: temporary mockup.
         pragma opaque;
+        aborts_if [abstract] false;
+        ensures [abstract] signer::address_of(result) == addr;
+    }
+
+    spec initialize {
+        aborts_if !system_addresses::is_aptos_framework_address(signer::address_of(aptos_framework));
+        aborts_if exists<OriginatingAddress>(signer::address_of(aptos_framework));
+        ensures exists<OriginatingAddress>(signer::address_of(aptos_framework));
+    }
+
+    spec create_account {
+        pragma aborts_if_is_partial;
```

Comment:
`aborts_if_partial` is almost always undesired, because it basically means the existing aborts_if clauses are useless, since there can be an arbitrary number of other abort conditions. We can do it if is temporary. In this case please add a TODO to later complete the aborts spec.

Reply:
Done